qemu/tests/qemu_guest_agent.py

```diff
@@ -126,11 +141,68 @@ def _check_ga_service(self, session, cmd_check_status):
         :param session: use for sending cmd
         :param cmd_check_status: cmd to check if ga service is started
         '''
-        error_context.context("Check whether qemu-ga service is started.", logging.info)
+        error_context.context("Check whether qemu-ga service is started.",
+                              logging.info)
         s, o = session.cmd_status_output(cmd_check_status)
         return s == 0
 
     @error_context.context_aware
+    def _get_latest_pkg(self):
+        """
+        get latest qemu-guest-agent rpm package.
+        :return: rpm pkg list
+        """
+        virt_module_stream = self.params.get("virt_module_stream", "")
+        guest_name = self.params.get("guest_name")
+        brew_url = "http://download.devel.redhat.com"
```

Comment:
It should not be provided upstream.

Reply:
update, could you have a check?

Reply:
In fact, I don't know what you want to do, so I will do a simple review.
Also, I will delete this comment as it exposes internal url.